java-client/src/generated/java/org/opensearch/client/opensearch/snapshot/ShardsStats.java

```diff
@@ -30,24 +30,34 @@
  * GitHub history for details.
  */
 
+//----------------------------------------------------
+// THIS CODE IS GENERATED. MANUAL EDITS WILL BE LOST.
+//----------------------------------------------------
+
 package org.opensearch.client.opensearch.snapshot;
 
 import jakarta.json.stream.JsonGenerator;
 import java.util.function.Function;
+import javax.annotation.Generated;
+import javax.annotation.Nonnull;
 import org.opensearch.client.json.JsonpDeserializable;
 import org.opensearch.client.json.JsonpDeserializer;
 import org.opensearch.client.json.JsonpMapper;
 import org.opensearch.client.json.ObjectBuilderDeserializer;
 import org.opensearch.client.json.ObjectDeserializer;
 import org.opensearch.client.json.PlainJsonSerializable;
 import org.opensearch.client.util.ApiTypeHelper;
+import org.opensearch.client.util.CopyableBuilder;
 import org.opensearch.client.util.ObjectBuilder;
 import org.opensearch.client.util.ObjectBuilderBase;
+import org.opensearch.client.util.ToCopyableBuilder;
 
-// typedef: snapshot._types.ShardsStats
+// typedef: snapshot.ShardsStats
 
 @JsonpDeserializable
-public class ShardsStats implements PlainJsonSerializable {
+@Generated("org.opensearch.client.codegen.CodeGenerator")
+public class ShardsStats implements PlainJsonSerializable, ToCopyableBuilder<ShardsStats.Builder, ShardsStats> {
```

Comment:
On the similar subject, may be:

```suggestion
public class SnapshotShardsStats implements PlainJsonSerializable, ToCopyableBuilder<ShardsStats.Builder, ShardsStats> {
```